### DISCUSSION TITLE
Porting CIRCLE to other SoC platforms

### DISCUSSION THREAD

**Discussion by Colpocorto** [rsta2/circle#327] [Ideas]
Hi,

maybe it's a naive question but I've searched the forum and haven't found anything about this.

Is there any chance that Circle is ported to the Orange Pi or other SoC systems?
Now that it is virtually impossible to get a RPI at a reasonable price (something that was intended to be the main reason of the existence of the Raspberry Pi itself), other SoC boards should be considered IMHO.

The Orange Pi boards are much cheaper than the Raspberry PI on Aliexpress and other sites. For instance, Orange Pi 5 (8GB of RAM / 8x 64bit cores) is about 90€:

https://www.aliexpress.com/item/1005004941850323.html

I understand that hardware is quite different, yet ARM-based though. different I/O options, less GPIO pins, etc. But maybe a fork devoted to the Orange Pi might be created.

Any thoughts?

**Comment by rsta2** [comment]
Yes, unfortunately currently it's very difficult to buy Raspberry Pis. I hope this situation will improve in next year.

One good thing of the different Raspberry Pi models is, that much of their hardware and firmware (!) is quite similar, which means, one does not have to write new drivers for all devices again and again. And there are many different devices in the SoC and out of it. This is an important reason, why Circle only works on Raspberry Pis.

I needed some focus to maintain this project over the time, and that focus is the family of Raspberry Pi computers. I will watch, how the supply situation will develop and never say never, but if it improves in a year or so, maybe with a new Raspberry Pi 5 (?), I will stay with this family of computers.

**Reply by Colpocorto** [reply]
Thank you for your response!
